### PR TITLE
Fix #21 : Project files are visible in the project tree

### LINKED ISSUE #21: Header and C files are not visible in project tree
When new files are copied from old directories to the `PSLab_Original`, they are not visible in the project tree.

### PR DESCRIPTION
**Pull request by CloudyPadmal**
Project files were manually added to the `configuration.xml` to make them visible in the Project Tree.

### CODE REVIEW

File: PSLab_Original/PSLAB_UART.c [hunk 1]
```diff
@@ -91,9 +94,7 @@ char getChar() {
 }
 
 unsigned int getInt() {
-    c1 = getChar()&0xFF;
-    c2 = getChar()&0xFF;
-    return (c2 << 8) | c1;
+    return ((getChar()&0xFF << 8) | (getChar()&0xFF));
```
Comment (jithinbp) [inline PSLab_Original/PSLAB_UART.c]: Lines 94 to 96 have been condensed into Line 97 , but it looks like the byte order will be reversed.
In the original case, LSB is received first, followed by MSB . In the new instance, the first BYTE will be treated
as MSB, and the second as LSB . Please confirm the order.

File: PSLab_Original/PSLAB_UART.c [hunk 2]
```diff
@@ -137,9 +138,7 @@ char getChar2(void) {
 }
 
 unsigned int getInt2(void) {
-    c1 = getChar2()&0xFF;
-    c2 = getChar2()&0xFF;
-    return (c2 << 8) | c1;
+    return ((getChar2()&0xFF << 8) | (getChar2()&0xFF));
```
Comment (jithinbp) [inline PSLab_Original/PSLAB_UART.c]: please refer to the previous comment on line 96 , function `unsigned int getInt`

File: PSLab_Original/PSLAB_UART.c (hunk 1 above)
Comment (CloudyPadmal) [inline PSLab_Original/PSLAB_UART.c]: Yes @jithinbp that would be wrong then! I'll fix it now! Thanks for pointing out!!

File: PSLab_Original/PSLAB_UART.c (hunk 2 above)
Comment (CloudyPadmal) [inline PSLab_Original/PSLAB_UART.c]: Thank you @jithinbp ! I've fixed it and sent a fix now!

**Comment by CloudyPadmal** [comment]
@jithinbp I've added `PSLab_Original/nbproject/private` to the `.gitignore` file! :+1:

**Comment by CloudyPadmal** [comment]
Changes were merged when #23 was merged. Hence closing this.

**Pull request closed without merging**